gateway_lambda/config.ts

```diff
@@ -28,7 +28,7 @@ const config = {
         EVENT.PROFILE_UPDATE,
       'https://schemas.accounts.firefox.com/event/delete-user':
         EVENT.USER_DELETE,
-      'https://schemas.accounts.firefox.com/event/apple-migration':
+      'https://schemas.accounts.firefox.com/event/apple-user-migration':
```

Comment:
Neither of these urls are accessible AFAICT. How do I know which one is right?

Reply:
@efixler nope, these are schemea ids provided by fxa.

Reply:
Ok @bassrock sorry if this is a dumb question, but don't these urls need to be be accessible?

Reply:
No it doesn't they use them as an ID in the payload they send us. 

We should set up time and i can walk you through it. This repo is purely for web hooks https://mozilla.github.io/ecosystem-platform/relying-parties/tutorials/integration-with-fxa#webhook-events